.github/actions/frontend/npm/publish/action.yml

```diff
@@ -13,11 +13,8 @@ runs:
       shell: bash
       run: |
         echo "auto_token=${{ inputs.auto_token }}"
-    - name: add_auto_to_globals
-      run: npm install -g auto@10.37.4
-      shell: bash
     - name: deploy_to_npm
       env:
         GH_TOKEN: ${{ inputs.auto_token }}
-      run: npx auto shipit
+      run: npx --yes -p auto@latest auto shipit
```

Comment:
@Sivli-Embir Addresses [this comment](https://github.com/JupiterOne/.github/pull/90#discussion_r1304922430)

Reply:
The only thing about this is, since its an external deps, it might be best to major version lock this.